pkg/clicmd/registry_add.go

```diff
@@ -77,22 +60,44 @@ SCM configuration.
 * ` + "`ks registry list` " + `— ` + regShortDesc["list"] + `
 
 ### Syntax
-`,
-	Example: `# Add a registry with the name 'databases' at the uri 'github.com/example'
+`
+	registryAddExample = `# Add a registry with the name 'databases' at the uri 'github.com/example'
 ks registry add databases github.com/example
 
 # Add a registry with the name 'databases' at the uri
 # 'github.com/example/tree/master/reg' and the version (branch name) 0.0.1
 # NOTE that "0.0.1" overrides the branch name in the URI ("master")
-ks registry add databases github.com/example/tree/master/reg --version=0.0.1`,
-}
+ks registry add databases github.com/example/tree/master/reg --version=0.0.1`
+)
 
-func init() {
-	registryCmd.AddCommand(registryAddCmd)
+func newRegistryAddCmd(a app.App) *cobra.Command {
+	registryAddCmd := &cobra.Command{
+		Use:     "add <registry-name> <registry-uri>",
```

Comment:
probably not worth the work it would take, but it could potentially be nice to have the usage text pulled out along side the descriptions and examples so that if something wants the usage it doesn't have to create a new cobra.Command to get it.